scormxblock/scormxblock.py

```diff
@@ -492,6 +493,19 @@ def _publish_grade(self, status, score):
                     'max_value': self.weight,
                 })
 
+    def publish_progress(self, scorm_data):
+        """
+        Mark 100% completion if course is complete
```

Comment:
the docstring here belongs more to the line inside the function i think; the method can be used to publish any percentage of completion not just 100